default.yml

```diff
@@ -47,6 +53,15 @@ Metrics/LineLength:
 Metrics/ModuleLength:
   Max: 200
 
+Metrics/ClassLength:
+  Max: 200
+
+Metrics/MethodLength:
+  Max: 20
```

Comment:
How about 15 lines? 20 feels too long for me, what does everyone else think?

Reply:
I agree that 20 seems long.

Reply:
I originally tried 15 but there were still some controller actions that it didn't seem to me were reasonable to break up into smaller methods. I suppose I'm willing to compromise on this issue a bit though.

Reply:
I still think 20 is better, but have made it 15 for now.